docs/changelog/4_x.rst

```diff
@@ -0,0 +1,473 @@
+====
+v4.0
+====
+
+:edb-alt-title: EdgeDB v4
+
+This release cycle is much shorter than the previous ones. It reflects our new
+approach at EdgeDB where the goal is to provide improvements at a steady
+regular pace rather than in big, but infrequent batches. Going forward we
+expect to maintain this shorter release cadence focusing on a few features at
+a time.
+
+To play with the new features, install the CLI using `our installation guide
+<https://www.edgedb.com/install>`_ and initialize a new project.
+
+.. code-block:: bash
+
+  $ edgedb project init --server-version 4.0-beta.1
+
+
+Upgrading
+=========
+
+**Local and Cloud instances**
+
+To upgrade a local project, first ensure that your CLI is up to date with
+``edgedb cli upgrade``. Then run the following command inside the project
+directory.
+
+.. code-block:: bash
+
+  $ edgedb project upgrade --to-testing
+
+Alternatively, specify an instance name if you aren't using a project.
+
+.. code-block:: bash
+
+  $ edgedb instance upgrade --to-testing -I my_instance
+
+**Hosted instances**
+
+To upgrade a remote (hosted) instance, we recommend the following
+dump-and-restore process.
+
+1. EdgeDB v4.0 only supports PostgreSQL 14 (or above). So check the version of
```

Comment:
I don't know if "upgrade postgres first" is the right advice.
I think the main thing is "your new 4.0 instance must point to postgres 14 or 15"